core/input/input_event.h

```diff
@@ -452,49 +452,79 @@ class InputEventGesture : public InputEventWithModifiers {
 	GDCLASS(InputEventGesture, InputEventWithModifiers);
 
 	Vector2 pos;
+	int touches;
 
 protected:
 	static void _bind_methods();
 
 public:
 	void set_position(const Vector2 &p_pos);
 	Vector2 get_position() const;
+	void set_touches(const int p_touches);
+	int get_touches() const;
+
+	InputEventGesture();
 };
 
-class InputEventMagnifyGesture : public InputEventGesture {
-	GDCLASS(InputEventMagnifyGesture, InputEventGesture);
-	real_t factor = 1.0;
+class InputEventGesturePan : public InputEventGesture {
+	GDCLASS(InputEventGesturePan, InputEventGesture);
+	Vector2 relative;
+	Vector2 speed;
 
 protected:
 	static void _bind_methods();
 
 public:
-	void set_factor(real_t p_factor);
-	real_t get_factor() const;
+	void set_relative(const Vector2 &p_relative);
+	Vector2 get_relative() const;
+
+	Vector2 get_delta() const;
+
+	void set_speed(const Vector2 &p_speed);
+	Vector2 get_speed() const;
 
 	virtual Ref<InputEvent> xformed_by(const Transform2D &p_xform, const Vector2 &p_local_ofs = Vector2()) const override;
 	virtual String as_text() const override;
-	virtual String to_string() override;
```

Comment:
Was this removal intended, or a rebase mishap? Other InputEvent classes seem to define `to_string()` too.